frontend/web/components/ButterBar.tsx

```diff
@@ -109,6 +122,21 @@ const ButterBar: React.FC<ButterBarProps> = ({ billingStatus, projectId }) => {
             payment method to keep your subscription active.
           </div>
         )}
+      {!!apiOrganisationUsage?.results.length && (
+        <div className='butter-bar'>
+          {'Your organisation has exceeded'}{' '}
+          <b>{apiOrganisationUsage?.results[0].percent_usage}</b>
+          {'% '}
```

Comment:
Tiny nit: the `%` should be within the bold tag, since it looks weird to bold just the number without the percent.

Reply:
Done, also I added the flag name in the description PR.